tools/RAiDER/gnss/processDelayFiles.py

```diff
@@ -224,6 +367,20 @@ def create_parser():
         default='Combined_delays.csv'
     )
 
+    p.add_argument(
+        '--localtime',
```

Comment:
The explanation was a little opaque to me. Rather than specify the details about 0 longitude, perhaps just state that the user can pass a localtime?

Reply:
Yeah, maybe a bit too superfluous in the parser. How about something like this instead:
"Optional control to pass data at specified, locally-projected time (1st argument) and pass data within specified +/- X hour threshold of local-time (2nd argument).
By default UTC is passed as is without local-time conversions.
Input in 'HH:MM:SS H', e.g. '16:00:00 1'""